modules/customize-certificates-access-default-service-serving-configmap.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/service-serving-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="access-default-service-serving-configmap_{context}"]
+= Access the service CA bundle in the default config map
+
+A pod can access the service CA certificate by mounting the default
+`ConfigMap` object that exists in every project. The service CA
+certificate is in the `service-ca.crt` key of the `openshift-service-ca.crt`
+config map. Access to this CA certificate allows TLS clients to verify
+connections to services using service serving certificates.
+
+.Procedure
+
+. View the config map to ensure that the service CA bundle is available:
++
+[source,terminal]
+----
+$ oc get configmap openshift-service-ca.crt -o yaml
```

Comment:
Probably needs a namespace set here?

Reply:
Yes, it would technically be better to use a specific namespace, but if you don't specify any, it returns the one on the default namespace / current project.